components/taggroup/stories/taggroup.stories.js

```diff
@@ -23,31 +25,88 @@ export default {
 			else value.table = { ...value.table, category: "Tag settings" };
 			return { ...acc, [key]: value };
 		}, {}),
-		ariaLabel: {
-			name: "Aria-label",
-			type: { name: "string" },
+		isInvalid: {
+			...isInvalid,
+			description: "Displays help text below the tag group with invalid icon and styling.",
+			if: { arg: "helpText", neq: "" },
+		},
+		ariaLabel: { table: { disable: true } },
+		label: { table: { disable: true } },
+		items: { table: { disable: true } },
+		actionButtonText: {
+			name: "Action button text",
+			description: "Displays an action button below the tag group, if left blank, the action button will not be displayed.",
+			type: { name: "text" },
+			table: {
+				type: { summary: "text" },
+				category: "Action button settings",
+			},
+			control: "text",
+		},
+		hasDisabledActionButton: {
```

Comment:
There's now a `hasDisabledActionButton` control and a new category for action button settings. We _could_ do more with the action button, let me know if you think that's valuable.